internal/db/migrations/011_delete_cascade.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE reservation_instances
+DROP CONSTRAINT reservation_instances_reservation_id_fkey,
+ADD CONSTRAINT reservation_instances_reservation_id_fkey
+FOREIGN key (reservation_id) REFERENCES reservations(id) ON DELETE CASCADE;
+
+ALTER TABLE aws_reservation_details
+DROP CONSTRAINT aws_reservation_details_pubkey_id_fkey,
+ADD CONSTRAINT aws_reservation_details_pubkey_id_fkey
+FOREIGN key (pubkey_id) REFERENCES pubkeys(id) ON DELETE CASCADE;
```

Comment:
Missing newline (small nit).

But wait with merging please, we also have GCP detail table and I want to check it.